trade-module/src/main/java/io/edpn/backend/trade/domain/controller/v1/TradeModuleController.java

```diff
@@ -11,4 +16,7 @@ public interface TradeModuleController {
 
     @GetMapping("/best-price")
     List<CommodityMarketInfoResponse> getBestCommodityPrice();
+
+    @PostMapping("/commodity")
```

Comment:
maybe rename to "locate-commodity"

"commodity" feels like it should be reserved for retrieving a list of commodities